storage/range.go

```diff
@@ -1575,13 +1581,12 @@ func (r *Range) loadFirstAndLastIndex() error {
 			return err
 		}
 		atomic.StoreUint64(&r.lastIndex, lastEnt.Index)
-
-		var firstEnt raftpb.Entry
-		err = gogoproto.Unmarshal(kvs[len(kvs)-1].Value.Bytes, &firstEnt)
+	} else if len(kvs) == 0 {
```

Comment:
Definitely throw in a comment for the len(kvs) == 0 case, explaining the two possibilities.

1. all logs truncated to nothing
2. new range with no logs yet